src/olympia/addons/tasks.py

```diff
@@ -435,6 +435,7 @@ def add_dynamic_theme_tag(ids, **kw):
         files = addon.current_version.all_files
         if any('theme' in file_.webext_permissions_list for file_ in files):
             Tag(tag_text='dynamic theme').save_tag(addon)
+            index_addons.delay([addon.id])
```

Comment:
You can do that after the loop, with `index_addons.delay(ids)`. Fewer tasks triggered that way.

Reply:
that would reindex _all_ public webextensions, rather than just the ~100 addons that are getting the new tag.

Reply:
Oh hey, yeah. I guess it's not worth trying to build a list of ids we changed, since there will not be many add-ons concerned by this anyway.